react/ProfileForm.tsx

```diff
@@ -323,6 +325,7 @@ const ProfileForm: React.FC = () => {
       if (profileUpdateSuccess && clientPreferencesUpdateSuccess) {
         // should go to the next step. maybe call a function exposed
         // in the checkout-container context
```

Comment:
As the issue was solved, I think this could be deleted.

Reply:
I was thinking whether or not I should remove this comment. I didn't because in the future we might still change this to a function inside the `checkout-container`, but yeah, I'll remove it